server/src/models/PayloadModel.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  CollectionOf,
+  Default,
+  Description,
+  Example,
+  getJsonSchema,
+  Required,
+  Title,
+} from '@tsed/common'
+import { NoteModel } from './NoteModel'
+
+// TODO: Edit model in accordance to the DB and API data
+
+enum Categories {
+  CAT1 = 'cat1',
+  CAT2 = 'cat2',
+}
+
+export class PayloadModel {
```

Comment:
What this model is responsible for?

Reply:
This is a boilerplate for the request payloads if they'll be different from the Note(s) model or include additional information. It's also TBD.